lib/vacuum.rb

```diff
@@ -0,0 +1,53 @@
+class Vacuum
```

Comment:
so I like this name, but it is also general enough that it isn't really clear to me what it does without opening it. Thoughts on more descriptive names?